docs/_writing-tests/rendering.md

```diff
@@ -84,8 +84,14 @@ Obviously, none of this can be taken as absolute rules, as testing of
 color-related functionality will necessitate using some of these
 colors!
 
+## Self Describing
+
+Tests should described the situations when they and when they fail. Read more about it [here][].
+
```

Comment:
```
+ Tests should should make it obvious when they pass and when they fail. Read more about it [here][].
```

I suggest change that wording to this:

*Tests should make it obvious when they pass and when they fail. It shouldn't be necessary to consult the specification to figure out whether a test has passed of failed.*

…and completely remove the *Read more about it [here][]* part — because those two sentences above are the entire verbatim contents of what’s at https://github.com/web-platform-tests/wpt/blob/master/docs/_writing-tests/general-guidelines.md#be-self-describing (so the guidance is short enough to be copied over to here entirely, so there’d no longer be any need to link to the other file).

And to be clear, IMHO at least it’s not bad to have the exact same text verbatim in two places in the docs. It’s all about context and providing readers with useful information at any point of use where they might need it — and readers might have use for the same information in multiple places.

Reply:
That makes sense to me. Thank you.
Made the requested changes.

Please take a look.